### PR TITLE
Correct name of home directory

### LINKED ISSUE #64: Leverage CISA_HOME environment variable in Python code
# 💡 Summary #

We should leverage the `CISA_HOME` environment variable in the Python code instead of hardcoding `/home/cisa`.

## Motivation and context ##

If the user overrides the `CISA_HOME` variable when building the Docker image then the code as it currently stands will fail.

## Implementation notes ##

The modification would take place [here](https://github.com/cisagov/saver/blob/9fba8f5b34de93d0a24fa758683b8ac7e7f8e956/src/pshtt_csv2mongo.py#L14-L15), for example.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request:
- Corrects the name of the home directory in several places (`/home/saver` :arrow_right: `/home/cisa`)
- Leverages the `CISA_HOME` environment variable in a few places
- Tidies up some double quotes in a shell script

## 💭 Motivation and context ##

- The first bullet above is necessary for the Docker container to function as expected
- The other bullets are some cleanup I went ahead and took care of while I was fixing the issue from the previous bullet
- Also resolves #64

## 🧪 Testing ##

All automated tests pass.  I also used these changes in CyHy production to successfully save the data from the most recent BOD 18-01 scanning run.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍

File: src/create_sld_to_agency_name_and_id_mapping.py [hunk 1]
```diff
@@ -11,8 +11,10 @@
 import yaml
 
 DB_CONFIG_FILE = "/run/secrets/scan_write_creds.yml"
-INCLUDE_DATA_DIR = "/home/saver/include/"
-SHARED_DATA_DIR = "/home/saver/shared/"
+# TODO: Leverage the CISA_HOME environment variable.  See #64 for more
```
Comment (mcdonnnj) [inline src/create_sld_to_agency_name_and_id_mapping.py]: Is there something stopping us from doing this now using the [`os` library](https://docs.python.org/3/library/os.html#os.environ)?

**Review by mcdonnnj — APPROVED** [review]
Ack I forgot to check this when reviewing #50. LGTM but I had one question.

File: src/create_sld_to_agency_name_and_id_mapping.py (hunk 1 above)
Comment (jsf9k) [reply src/create_sld_to_agency_name_and_id_mapping.py]: I was just leery of making more changes after I had just tested the code in production this morning.  I can make the change now.

Please see commit 1b9bac8.

Comment (mcdonnnj) [reply src/create_sld_to_agency_name_and_id_mapping.py]: Sorry I just thought it would be best to do this together logistically. Thanks for updating!

**Pull request merged**